src/main/resources/jenkins/security/plugins/ldap/Messages.properties

```diff
@@ -1,2 +1,50 @@
 FromUserRecordLDAPGroupMembershipStrategy.DisplayName=Parse user attribute for list of groups
-FromGroupSearchLDAPGroupMembershipStrategy.DisplayName=Search for groups containing user
\ No newline at end of file
+FromGroupSearchLDAPGroupMembershipStrategy.DisplayName=Search for groups containing user
+LDAPSecurityRealm.LoginHeader=Login
+LDAPSecurityRealm.AuthenticationSuccessful=Authentication: successful
+LDAPSecurityRealm.AuthenticationFailed=Authentication: failed for user "{0}"
+LDAPSecurityRealm.AuthenticationFailedEmptyPass=Authentication: failed for user "{0}" with empty password
+LDAPSecurityRealm.UserId=User ID: {0}
+LDAPSecurityRealm.UserDn=User Dn: {0}
+LDAPSecurityRealm.UserDisplayName=User Display Name: {0}
+LDAPSecurityRealm.UserEmail=User email: {0}
+LDAPSecurityRealm.GroupMembership=Group membership:
```

Comment:
Isn't it only the "LDAP group membership"? The code is skipping the AUTHENTICATED_AUTHORITY group.